qemu/tests/cfg/rng_bat.cfg

```diff
@@ -12,6 +12,13 @@
         read_rng_cmd  = ${rng_dst}
         driver_name = "viorng"
         rng_data_rex = "0x\w"
+        cdroms += " virtio"
+        i386:
+            devcon_dirname = "x86"
+        x86_64:
+            devcon_dirname = "amd64"
+    	devcon_path = "WIN_UTILS:\devcon\${devcon_dirname}\devcon.exe"
+        need_memory_leak_check = yes
```

Comment:
Recommended to define it with the same parameter `memory_leak_check`.

Reply:
some cases need call 'rng_bat'  as sub_test ，so we have to use a different parameter 'need_memory_leak_check'  in driver_in_use.py to avoid some unnessary  steps.